benchmarks/osb/procedures/no-train-test.json

```diff
@@ -46,6 +46,16 @@
                 "retries": 100
             }
         },
+        {
+            "operation": {
+                "name": "force-merge",
+                "operation-type": "force-merge",
+                "request-timeout": {{ target_index_force_merge_timeout }},
+                "index": "{{ target_index_name }}",
+                "mode": "polling",
```

Comment:
Any special reason on why it's "polling"? Should we maybe say something in readme if we recommend polling mode?

Reply:
This is actually a OpenSearch benchmarks operation. Their documentation should cover this. I dont want to add documentation about their operations in our README because it will be hard to draw the line. This parameter signals that if the request timeouts, it should retry.